scripts/log_reports.py

```diff
@@ -26,8 +26,7 @@
                         total_num_failed += 1
                     else:
                         passed.append([test, duration, log.name.split('_')[0]])
-        if nb_lines == 0:
-            empty_file = True
+        empty_file = i == 0
```

Comment:
Hmm, when there is exactly one line,`i` would also be 0. How about inside of the loop, doing `i += 1` instead of using enumerate?